package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "8.1.0",
+  "version": "8.2.0",
```

Comment:
would this qualify as a major release (going to `9.0.0`)?

Reply:
@erluti yes this is a breaking change and major version number should increase.  I will make it so thanks!